test/services/workflow_executions/status_service_test.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module WorkflowExecutions
+  class CreateServiceTest < ActiveSupport::TestCase
+    def setup
+      @user = users(:john_doe)
+      @workflow_execution = workflow_executions(:irida_next_example_prepared)
+    end
+
+    test 'get status of workflow execution which has completed' do
+      stubs = Faraday::Adapter::Test::Stubs.new
+      stubs.post('/runs') do
+        [
+          200,
+          { 'Content-Type': 'application/json' },
+          { run_id: 'abc123' }
```

Comment:
Nitpick: We should use different run id's for the different test files, and maybe for each file too. Something like `status_test_1`

Reply:
Yeah good idea. Updated in 93a0a6b8e0a98ae38079d25f69a559f3943669d7